changelog

```diff
@@ -1,3 +1,400 @@
+      * Ships with the "staying" personality can no longer land or leave the system if they also have the "surveillance" personality. (@warp-core)
+      * Fixed a null pointer dereference in the MapPanel. (@warp-core)
+      * Out of system mission cargo and passengers will no longer be teleported to the player's location when loading a save. (@warp-core)
+      * It is no longer possible to select invalid systems in the map using keyboard shortcuts. (@Hurleveur)
+      * Don't try to parse the "display name" token in an outfit like a normal attribute. (@warp-core)
+      * Ensured some primitive variables are properly initialized. (@Hurleveur, @petervdmeer)
+      * Fixed the minimum distance calculation for departure distance. (@wjp)
+      * Don't try to load non-existedn plugin icons. (@quyykk)
+      * Ensured ships will always have an attraction and deterrence value. (@quyykk)
+      * Ensured options in conversation choices which aren't being displayed can't be selected with keyboard navigation. (@UnorderedSigh)
+      * Improved behaviour of out of system escorts without fuel. (@wjp)
+      * Fixed a null pointer dereference in the player info panel when the player has multiple ships but no flagship. (@warp-core)
+      * Do not offer landing missions in the shipyard or outfitter. (@warp-core)
+      * Do not allow disabled ships to dock with carriers. (@UnorderedSigh)
+      * Lines in interfaces now use the centre of the defining rectangle instead of the top left corner. (@thewierdnut)
+      * Fixed various issues reported by Cppchecks. (@mike-f1)
+      * Less than 1 unit of free cargo space is no longer rounded down when attempting to buy an outfit into cargo. (@warp-core)
+      * Fixed a crash due to a nullpointer dereference in the parallax code when the player has no flagship. (@warp-core)
+      * Fixed a vector iterator incompatibility in ShipInfoDisplay. (@warp-core)
+      * Park off-system escorts that fail flight or bunk checks. (@yjhn, @Amazinite)
+  * Game content:
+    * New content:
+      * Two new human guns: Twin Blaster and Modified Twin Blaster. (@Daeridanii1)
+      * An easter egg if the pilot has a certain name. (@RisingLeaf)
+      * Additional Hai names and hail phrases. (@MasterOfGrey)
+      * A person ship for Gefüllte Taubenbrust. (@Saugia)
+      * Two new Hai transport ships: the Cicada and Scarab. (@Saugia)
+      * A new ship and some outfits for the Unfettereed. (@beccabunny, @EjoThims, @Hurleveur)
+        * The Sea Scorpion: a medium warship.
+        * Tripulse Shredder.
+        * Ionic Blaster and Turret weapons.
+        * Value Detector: a new scanner.
+      * Two new ships for the Korath Exiles. (@ravenshining, @Saugia)
+        * Rai'alorej: a search and rescue hospital ship.
+        * Ikatila'ej: a large mining and manufacturing ship with a unique spinal weapon.
+      * A surveillance drone for the Korath Exiles: the Kas'ik Tek 7. (@Saugia)
```

Comment:
```suggestion
        * Kas'ik Tek 7: a surveillance drone for the Korath Exiles.
```
To match above, but double check that spacing though, just in case I screwed it up.